src/Core/src/Handlers/Layout/LayoutHandler.Windows.cs

```diff
@@ -123,12 +123,14 @@ void EnsureZIndexOrder(IView child)
 			}
 		}
 
-		static void MapInputTransparent(ILayoutHandler handler, ILayout layout)
+		public static partial void MapBackground(ILayoutHandler handler, ILayout layout)
 		{
-			if (handler.PlatformView is LayoutPanel layoutPanel && layout != null)
-			{
-				layoutPanel.UpdatePlatformViewBackground(layout);
-			}
+			handler.PlatformView?.UpdatePlatformViewBackground(layout);
+		}
+
+		public static partial void MapInputTransparent(ILayoutHandler handler, ILayout layout)
+		{
+			handler.PlatformView?.UpdatePlatformViewBackground(layout);
 		}
```

Comment:
Seems Windows does IT right, but then the background just wipes it all out. Fixed this by making Background call the correct method.